app/models/report_generators/system_performance/fy2019/measure_one.rb

```diff
@@ -516,8 +516,12 @@ def filter_days_for_homelessness dates, project_types, stop_project_types
       (! stop_project_types.include?(night[:project_type]))
     end
 
-    private def in_stop_project_on?(night, date, stop_project_types)
-      (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].blank? || night[:MoveInDate] <= date))
+    private def in_stop_project_on?(night, date, stop_project_types, consider_move_in_dates)
+      if consider_move_in_dates && PH.include?(night[:project_type])
+        return (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].present? && night[:MoveInDate] <= date))
+      else
+        return (stop_project_types.include?(night[:project_type]) && (night[:MoveInDate].blank? || night[:MoveInDate] <= date))
```

Comment:
These should be  `night[:MoveInDate] < date`

Reply:
I think I was wrong before...I think `<=` is correct, that way it return returns true on the move in date.